cmd/edit/machinepool/cmd.go

```diff
@@ -236,6 +236,11 @@ func run(cmd *cobra.Command, argv []string) {
 			os.Exit(1)
 		}
 	}
+
+	if labels == "\"\"" {
```

Comment:
What if we apply a transformer on the survey input instead? Not sure if we ever want an escaped empty string when prompting anything interactively

https://github.com/AlecAivazis/survey/blob/master/transform_test.go

https://github.com/openshift/rosa/blob/ab250b83b4bafceaa1c91cba89d2eef2ce29e536/pkg/interactive/interactive.go#L60

Reply:
Good point, could someone let me know if this is the best path forward? I will make this change if so.